### PR TITLE
arch/imxrt/adc: set ADC with  highest sampling rate config

### PR DESCRIPTION
**Pull request by an4967**
- set ADC with higest sampling rate config

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [1dbee26eb8a819e910b6a4bca89d5defeabca284] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [1dbee26eb8a819e910b6a4bca89d5defeabca284] - Code Rule Check OK.

File: framework/src/iotbus/iotbus_adc.c [hunk 1]
```diff
@@ -297,7 +297,7 @@ uint32_t iotbus_adc_get_sample(iotbus_adc_context_h hnd, int timeout)
 	readsize = sizeof(struct adc_msg_s);
 
 	handle->state = IOTBUS_ADC_BUSY;
-	ret = ioctl(handle->fd, ANIOC_TRIGGER, NULL);
+	ret = ioctl(handle->fd, ANIOC_TRIGGER, 0);
 	if (ret < 0) {
 		ibdbg("[ADC] trigger error(%d)\n", ret);
 		handle->state = IOTBUS_ADC_RDY;
```
Comment (davidfather) [inline framework/src/iotbus/iotbus_adc.c]: The return value should be a negative value from iotbus_error_e.

Comment (an4967) [reply framework/src/iotbus/iotbus_adc.c]: Thanks, I will fix it.

**Comment by seinfra** [comment]
Target : [a43813251ed3e22502743665f126955e2f6bf29d] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [a43813251ed3e22502743665f126955e2f6bf29d] - Code Rule Check OK.

**Pull request merged**